analysis/test_data_utils.py

```diff
@@ -69,6 +70,37 @@ def test_drop_uniteresting_columns():
     assert 'time_started' not in cleaned_df.columns
 
 
+def test_experiments_data():
+    """Tests that clobber experiments data clobbers stale snapshots from earlier
+    experiments."""
+    experiments = []
+    df = pd.DataFrame()
+    for experiment_num in range(3):
+        experiment = 'experiment-%d' % experiment_num
+        experiments.append(experiment)
+        experiment_df = create_experiment_data(experiment)
+        df = pd.concat([df, experiment_df])
```

Comment:
nit: can this be simplified with
`  df = pd.concat([create_experiment_data('experiment-' + n) for n in range(3)])
`?